third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -853,7 +853,6 @@ func TestAccContainerCluster_withNodeConfigShieldedInstanceConfig(t *testing.T)
 	})
 }
 
-<% unless version.nil? || version == 'ga' -%>
```

Comment:
This is a positive change (assuming the field(s) are GA) but not related to the rest of the PR.

Reply:
This might have been left over from a merge conflict I was solving - I know merging in the commit that made this GA had some conflicts that I solved, and I did a search and replace for `WorkloadMetaDataConfig` to make sure I had everything GA - but it could be the original commit overlooked making this test GA and I just did it here. I can double-check to see what happened and pull it into a new PR, or just verify that it definitely should be GA and go with it? Whichever you feel is best!

Reply:
Verifying it's GA is 👍  with me!

Reply:
Yup! Looks like the nodepool tests were properly moved to GA, but the cluster ones were missed https://github.com/GoogleCloudPlatform/magic-modules/pull/3941/files so we should be good here!